README.md

```diff
@@ -19,12 +19,31 @@ Include the package locally in your repository.
 
 `npm install dotenv-webpack --save`
 
+### Description
+
+`dotenv-webpack` wraps `dotenv` and `Webpack.DefinePlugin`. As such, it overwrites existing anyexisting `DefinePlugin` configurations. Also, like `DefinePlugin`, it does a text replace in the resulting bundle for any instances of `process.env`. 
+
+As such, if used with `React` in production, make sure to include `NODE_ENV=production` inside your `.env` file.
```

Comment:
While this is a common use-case, I don't think you would need to mention React explicitly.

Instead I would mention something as follows:

>Any script files that get bundled with webpack and reference `process.env` would need to use this plugin to access the variables defined in the `.env` file.